meta-lmp-bsp/recipes-security/optee/optee-os-fio-bsp-common.inc

```diff
@@ -3,6 +3,28 @@ OPTEE_HEADER:stm32mp1common    = "tee-header_v2"
 OPTEE_PAGEABLE:stm32mp1common  = "tee-pageable_v2"
 OPTEE_PAGER:stm32mp1common     = "tee-pager_v2"
 
+# TI HS signing logic (from meta-ti-bsp optee-os bbappend)
+do_compile:prepend:k3() {
+    export TI_SECURE_DEV_PKG=${TI_SECURE_DEV_PKG}
```

Comment:
This seems TI specif and don't see it defined anywhere because can't found references to TI_SECURE_DEV_PKG_K3
```
meta-ti/meta-ti-bsp/conf/machine/include/k3.inc:TI_SECURE_DEV_PKG = "${TI_SECURE_DEV_PKG_K3}"
```

Reply:
checked locally and the variable is not used nor defined
```
builder@9e4c0e2ef608:/lmp/build-am64xx-evm$ bitbake-getvar -r optee-os-fio TI_SECURE_DEV_PKG_K3
#
# $TI_SECURE_DEV_PKG_K3
#   [no history recorded]
#
#   "None"
```

Reply:
Yup, this is expected as the secure dev pkg is behind NDA, the user would need to download and set the variable in order to be able to use it.

Reply:
Understood. Thanks